docs/index.rst

```diff
@@ -19,3 +19,10 @@ The Software Factory documentation is the central space where we collect all use
     swf-blueprint/docs/articles/templates/index
     swf-blueprint/docs/articles/licensing/index
     swf-blueprint/docs/articles/intro/index
+
+.. toctree::
+    :caption: Categories:
```

Comment:
I'm not sure why you have the colon at the end here, I'd remove it to have it consistent with the other TOC.